.gitignore

```diff
@@ -7,3 +7,5 @@ config/*.js
 logs
 .env
 .DS_Store
+.idea/
```

Comment:
For personal files, such as editor configs, use global `~/.gitignore` of configure rules for specific dirs in `~/.gitconfig`